Resources/doc/reference/installation.rst

```diff
@@ -151,4 +151,27 @@ Enable the Timeline Block
 
 And now, you're good to go !
 
+Edit the Timeline Block
+-------------------------  
+
+.. configuration-block::
```

Comment:
why the configuration block here? this should be done before the yaml code block